GomokuServer/src/GomokuServer.Api/Configuration/AppConfigurator.cs

```diff
@@ -0,0 +1,30 @@
+namespace GomokuServer.Api.Configuration;
+
+public static class AppConfigurator
+{
+	public static void ConfigureApp(WebApplication app)
+	{
+		ConfigureSwagger(app);
+		ConfigureSecurity(app);
+
+		app.UseCors(CorsPolicyName.GomokuClient);
+
+		app.MapControllers();
+		app.MapHub<GameHub>("/gamehub");
+
+		app.Run();
+	}
+
+	private static void ConfigureSwagger(WebApplication app)
+	{
+		SwaggerConfigurator.ConfigureSwaggerUi(app);
+	}
+
+	private static void ConfigureSecurity(WebApplication app)
```

Comment:
Don't prefix method with Configure if you adding middlewares in it. Prefix with Use, to stick to framework convention